### DISCUSSION TITLE
Threat Events View based on Diagram

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#26] [Q&A] [answered]
I have created two diagrams and generated the threat events.  Is there any way I can filter the threat events for a specific diagram ?

**Comment by simonec73** [comment, marked as answer]
No, and it would not be possible because the threats are associated to the entities, not to the diagrams. This means that if an entity has a threat in a diagram, it has a threat also in a different diagram where it is included.
